danielMunteanu/src/main/java/danielCollectionTask/Item.java

```diff
@@ -0,0 +1,44 @@
+package danielCollectionTask;
+
+public class Item {
+    final long id;
+    final String title;
+    final Type type;
+    private long idCounter;
+    final double price;
+
+    public Item(String title, Type type, double price) {
+        this.id = ++idCounter;
```

Comment:
let's have the same pattern for ID generating -> go with UUID as examples above

Reply:
done with UUID